parquet/src/test/java/org/apache/iceberg/parquet/TestParquetMetrics.java

```diff
@@ -21,14 +21,19 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.util.Map;
+
```

Comment:
Imports should not have blank lines. Sorry there isn't a checkstyle rule running for this yet, we still need to update the build for this one.